lib/utils/validator.dart

```diff
@@ -1,13 +1,13 @@
 final RegExp _emailRegex = new RegExp(
     r'^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$');
 
-bool isEmail(email) {
-  return _emailRegex.hasMatch(email);
+bool isEmail(String? email) {
+  return _emailRegex.hasMatch(email ?? '');
```

Comment:
You could return false whenever email is null. Would avoid the regex call.

Reply:
That makes sense.